apps/prairielearn/src/migrations/20240301223248_course_requests__not_null.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE course_requests
+ALTER COLUMN approved_status
+SET NOT NULL;
+
+ALTER TABLE course_requests
+ALTER COLUMN short_name
+SET NOT NULL;
+
+ALTER TABLE course_requests
+ALTER COLUMN title
+SET NOT NULL;
+
+ALTER TABLE course_requests
+ALTER COLUMN user_id
+SET NOT NULL;
```

Comment:
I checked in all prod environments and these were all already non-null, so I figured I'd enforce it at the schema level. This makes some usage of these columns safer.